packages/validation/src/index.js

```diff
@@ -16,7 +15,21 @@ export default function validateAST(ast: Program) {
   }
 }
 
-export { isConst } from "./is-const";
+export function getValidationErrors(ast: Program): Array<string> {
+  const errors = [];
+
+  ast.body.filter(({ type }) => type === "Module").forEach(m => {
+    const moduleContext = moduleContextFromModuleAST(m);
+
+    errors.push(...isConst(ast, moduleContext));
+    errors.push(...importOrderValidate(ast));
+    errors.push(...typeChecker(ast, moduleContext));
+  });
```

Comment:
There are some inconsistencies in the way the `repl` constructs modules. It doesnt always pass a `Program` here, but sometimes `ast` is directly a `Module` node itself. I tried checking for it here, but it didnt fix all the tests, so it probably requires some changes to the repl?

Reply:
I found the problem. The repl also used the type checker directly. I switched to the new  `getValidationErrors` API.